Makefile.include

```diff
@@ -29,9 +28,9 @@ release:              ## Build release versions of all components
 	make -C qan-api2 release
 
 gen: clean            ## Generate files
-	make gen-api
-	make gen-alertmanager
-	make clean_swagger
+	make -C api gen
+	make -C api gen-alertmanager
+	make -C api clean_swagger
```

Comment:
Can we rename it to `clean-swagger`? It looks inconsistent :|

Reply:
Aha, done